cmd/rook/ceph/mgr.go

```diff
@@ -45,11 +45,16 @@ func init() {
 }
 
 func startMgr(cmd *cobra.Command, args []string) error {
-	required := []string{"mon-endpoints", "cluster-name", "mon-secret", "admin-secret", "public-ipv4", "private-ipv4"}
+	required := []string{"mon-endpoints", "cluster-name", "mon-secret", "admin-secret"}
 	if err := flags.VerifyRequiredFlags(mgrCmd, required); err != nil {
 		return err
 	}
 
+	renamed := [][2]string{{"public-ip", "public-ipv4"}, {"private-ip", "private-ipv4"}}
+	if err := flags.VerifyRenamedFlags(mdsCmd, renamed); err != nil {
```

Comment:
the `mdsCmd` is accidentally being checked here when you want to use the command that this file is for.  this issue exists for all the other commands.

Reply:
Thanks for your reminder， I corrected all the mistakes